lib/v2/hdu/grs.js

```diff
@@ -0,0 +1,81 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const dayjs = require('dayjs');
```

Comment:
Please use `parseDate` instead of `dayjs` directly.
docs: https://docs.rsshub.app/joinus/advanced/pub-date